qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
 
 .. _issues_unzip:
 
-How to solve unzip errors?
---------------------------
-
-When downloading large zip files within Qiita there is a change that you will get
+How to solve download or unzip errors?
+--------------------------------------
+
+Dealing with large files might be daunting but, in general, following these
+instructions should make things easier. First, make sure that you have enough space
+for the zip download file; for the American Gut, you should have at least close to 700Gb
```

Comment:
Would it be possible to, near the download link, just show the size of the zip file to download? The size estimate for the AGP will be incorrect pretty soon.

Reply:
Just noticed that the full uncompressed + the zip is close to 2T; will change. BTW estimating the size of the zip is actually pretty hard to do accurately for the merged zip files but we could do ballpark estimates; let's discuss in the Wednesday qiita meeting. BTW there is an issue about this: https://github.com/biocore/qiita/issues/2133